### DISCUSSION TITLE
unable to startNotifcations

### DISCUSSION THREAD

**Discussion by Theo-55** [capacitor-community/bluetooth-le#691] [General]
This may be a personal error but could really use some help. Im attempting to connect to an MCU supporting BLE. On web, i am able to connect to the device and get the id, capture services and characteristics. The problem arrises when attempting to start notifications. When attempting to start the notifications i am receiving a GATT error as well as an unsupported device error. The goal here is that have the notifications update on my frontend every time a sensor is hit. We are putting sensors on a putting mat to track putts made. Simple, yet proving to be difficult.

Expected behavior
I would expect to see just have basic console logs of the notifications when a sensor is hit.

` async startNotifications(deviceId: string, service: string, characteristic: string): Promise {
try {
const readValue = await BleClient.read(deviceId, service, characteristic);
console.log('Read value:', readValue);

    const decoder = new TextDecoder('utf-8');
    const message = decoder.decode(readValue);
    console.log('Parsed read message:', message);

    await BleClient.startNotifications(deviceId, service, characteristic, (value) => {
      console.log('Notification received:', value);
      const notificationMessage = decoder.decode(value);
      console.log('Parsed notification message:', notificationMessage);
    });

  } catch (error) {
    console.error(`didnt work :(`, error);
  }
}`

Plugin version:

@capacitor-community/bluetooth-le: [e.g. 0.4.0]
Desktop (please complete the following information):
Latest Chrome version

Additional context

**Comment by peitschie** [comment]
Hi @Theo-55 

My normal recommendation in this type of situation is try  [nrf Connect](https://play.google.com/store/apps/details?id=no.nordicsemi.android.mcp) (also available on iOS) and perform the steps you're wanting manually and see if there's any error there too.

If nrf Connect shows the same symptoms, it's generally something wrong with the Bluetooth device. If the behaviour is different in nrf Connect, then it's worth looking further at your code and the code in this plugin for possible causes.
